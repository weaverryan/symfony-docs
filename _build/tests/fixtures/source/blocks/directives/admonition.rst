.. admonition:: Screencast
    :class: screencast_class

    Do you prefer video tutorials? Check out the the screencasts.
