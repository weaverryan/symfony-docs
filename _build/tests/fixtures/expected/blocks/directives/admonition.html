<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-wrapper screencast_class">
    <div class="admonition admonition-">
        <p class="admonition-title">Screencast</p>
        <p>Do you prefer video tutorials? Check out the the screencasts.</p>
    </div>
</div>
</body>
</html>
